frontend/src/app/features/folios/redux/FolioSlice.ts

```diff
@@ -183,7 +183,12 @@ const folioSlice = createSlice({
         state.fetchRequestStatus = RequestStatus.failed;
       })
       .addCase(addFolioItem.fulfilled, (state, action) => {
-        if (action?.payload?.data?.addFolioItem?.httpStatusCode === 400)
+        if (
+          action?.payload?.data?.addFolioItem?.httpStatusCode === 400 ||
```

Comment:
pls use the HttpStatus class instead, eg: HttpStatus.NOT_FOUND

Reply:
Already fixed in previous PR